<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobsTabWidget</class>
 <widget class="QWidget" name="JobsTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>356</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="jobsTabWidgetVerticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="jobsHeader">
     <property name="styleSheet">
      <string notr="true">#jobsHeader {
background: rgba(246,246,246,255);
margin-bottom: -1px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="jobsHeaderLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="jobsCountLabel">
        <property name="text">
         <string>Jobs</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="jobsFilterButton">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
border: transparent;
padding: 2px;
}

QToolButton:checked {
border: 1px solid darkgrey;
}</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/icons/magnifying-glass.png</normaloff>:/icons/magnifying-glass.png</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+F</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="jobsFilterFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="jobsFilterFrameLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="jobsFilter">
        <property name="toolTip">
         <string>Search and filter the Jobs list, shell globbing patterns can be used &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
border: none;
padding-right: 6px;
padding-left: 6px;
}

QComboBox::drop-down {
border: none;
padding-right: 6px;
}

QComboBox::down-arrow {
image:url(:/icons/caret-bottom.png);
}

QComboBox::down-arrow:on {
top: 1px;
left: 1px;
}</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="jobsWidget" native="true">
     <layout class="QVBoxLayout" name="jobsWidgetVerticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="jobsSplitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="JobListWidget" name="jobListWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Job list</string>
         </property>
         <property name="styleSheet">
          <string notr="true">#jobListWidget::item
{
selection-background-color: rgba(177,214,255,150);
}</string>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="layoutMode">
          <enum>QListView::Batched</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="batchSize">
          <number>20</number>
         </property>
        </widget>
        <widget class="JobDetailsWidget" name="jobDetailsWidget" native="true">
         <property name="toolTip">
          <string>Job details</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="defaultJobs" native="true">
     <property name="styleSheet">
      <string notr="true">#defaultJobs {
background: rgba(64,152,212,255)
}

QLabel {
color: white;
}

QPushButton {
border: 1px solid white;
background: transparent;
color: white;
padding: 4px;
border-radius: 2px;
}

QPushButton:hover {
border: 1px solid grey;
}

QPushButton:pressed {
border: 1px solid grey;
background: white;
color: black;
}</string>
     </property>
     <layout class="QVBoxLayout" name="defaultJobsVerticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="defaultJobsLabel">
        <property name="text">
         <string>Would you like to automatically create a handful of predefined Jobs for commonly-used directories such as Desktop, Documents, and Pictures?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="defaultJobsButtonsWidget" native="true">
        <layout class="QHBoxLayout" name="defaultJobsButtonWidgetHorizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="dismissButton">
           <property name="text">
            <string>Dismiss</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="defaultJobsSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="sureButton">
           <property name="text">
            <string>Yes, create them</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addJobButton">
       <property name="toolTip">
        <string>Add a new Job &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
background: rgba(64,152,212,255);
border: 1px solid white;
border-radius: 2px;
color: white;
font-size: 12px;
padding-right: 24px;
width: 100px;
height: 20px;
}

QToolButton:hover {
border: 1px solid grey;
}

QToolButton:pressed {
border: 1px solid grey;
background: white;
color: black;
padding-left: 2px;
padding-top: 2px;
}

QToolButton:disabled
{
background: darkgrey;
}

QToolButton:focus
{
border: 1px solid grey;
background: white;
color: black;
}

QToolButton::menu-button
{
border-left: 1px solid white;
margin-top: 1px;
margin-bottom: 1px;
width: 16px;
margin-right: 4px;
}

QToolButton::menu-button:active
{
border-left: 1px solid grey;
}

QToolButton::menu-arrow:open {
top: 1px;
left: 1px;
}

QToolButton QMenu
{
text-align: center;
}</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="save" stdset="0">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddJob">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/hard-drive.png</normaloff>:/icons/hard-drive.png</iconset>
   </property>
   <property name="text">
    <string>Add Job</string>
   </property>
   <property name="toolTip">
    <string>Add a new job</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBackupAllJobs">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-upload-multi.png</normaloff>:/icons/cloud-upload-multi.png</iconset>
   </property>
   <property name="text">
    <string>Backup all Jobs</string>
   </property>
   <property name="toolTip">
    <string>Backup all jobs now</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionJobRestore">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-download.png</normaloff>:/icons/cloud-download.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore latest backup</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionJobInspect">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/info.png</normaloff>:/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>Inspect</string>
   </property>
   <property name="toolTip">
    <string>Show job details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionJobBackup">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/cloud-upload.png</normaloff>:/icons/cloud-upload.png</iconset>
   </property>
   <property name="text">
    <string>Backup</string>
   </property>
   <property name="toolTip">
    <string>Execute backup for job(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionJobDelete">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete job</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionFilterJobs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/magnifying-glass.png</normaloff>:/icons/magnifying-glass.png</iconset>
   </property>
   <property name="text">
    <string>Filter and search</string>
   </property>
   <property name="toolTip">
    <string>Search and filter Jobs list  &lt;span style=&quot;color:gray;font-size:small&quot;&gt;%1&lt;/span&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JobDetailsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/jobwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JobListWidget</class>
   <extends>QListWidget</extends>
   <header>widgets/joblistwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>jobsFilterButton</tabstop>
  <tabstop>jobsFilter</tabstop>
  <tabstop>jobListWidget</tabstop>
  <tabstop>dismissButton</tabstop>
  <tabstop>sureButton</tabstop>
  <tabstop>addJobButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
